src/main/java/logging/Logger.java

```diff
@@ -109,37 +111,38 @@ private synchronized void log(LogLevel level, String text)
 
     /**
      * Logs a byte array to verbose output
-     * @param bytes The byte array to log
+     * @param packet The packet to log
      */
-    public synchronized void logVerbose(byte[] bytes)
+    public synchronized void logVerbose(DatagramPacket packet)
```

Comment:
How do we feel about this change?

Reply:
Instead of replacing, it would be nice to have both. Sometimes when Im making changes or testing stuff out it's nice to print the byte array without needing a packet!